react-components/my-app/src/ImageContainer.tsx

```diff
@@ -0,0 +1,7 @@
+export function ImageContainer() {
+  return (
+    <>
+      <img src="../public/image.png" />
```

Comment:
If they're in the public folder you should just reference them as `src="/image.png"`.